### ISSUE TITLE
retraining on new dataset

### ISSUE THREAD

**Issue by GlastonburyC** [facebook/fb.resnet.torch#144]
I'm slightly unsure about the wording of this tutorial:
https://github.com/facebook/fb.resnet.torch/blob/master/pretrained/README.md#fine-tuning-on-a-custom-dataset

I have a new dataset with 8-labels. When I perform retraining and then use that model to predict the labels, they are ImageNet labels, and not my own labels. How do I get it to output my datasets labels?

**Comment by colesbury** [comment]
If you're using the `classify.lua` script for prediction, you'll need to update the list of labels if you want the correct text:

https://github.com/facebook/fb.resnet.torch/blob/master/pretrained/classify.lua#L20
https://github.com/facebook/fb.resnet.torch/blob/master/pretrained/imagenet.lua

**Comment by GlastonburyC** [comment]
Cheers. So I simply create my own labelset.lua and point classify.lua in the right direction?

**Comment by colesbury** [comment]
Yeah, exactly

**Comment by GlastonburyC** [comment]
Additionally, the extract-features.lua script,  can I obtain bounding box locations using this?

**Comment by colesbury** [comment]
No. There are some tricks you can try like making your image larger and
looking for "hot spots" in the activations, but it won't work very well. If
you want decent bounding boxes you'll want to use something like Fast R-CNN

**Comment by GlastonburyC** [comment]
Understood. i think I mis-read its purpose initially and the term feature. Thanks for the help!
